rust_src/src/lisp.rs

```diff
@@ -89,7 +89,11 @@ impl LispObject {
 
     #[inline]
     pub fn from_bool(v: bool) -> LispObject {
-        if v { unsafe { Qt } } else { Qnil }
+        if v {
+            unsafe { Qt }
+        } else {
+            Qnil
+        }
```

Comment:
FWIW I think the oneline version is the result of `rustfmt`, so will be reverted the next time we run rustfmt. No matter.